src/App/MessageHandler.php

```diff
@@ -25,7 +25,7 @@ class MessageHandler
     /**
      * @brief Lève une erreur bloquante.
      *
-     * @param Exception $exception Exception à afficher
+     * @param Exception $exception Exceptions à afficher
```

Comment:
Aïe le Refactor qui s'est planté
```suggestion
     * @param Exception $exception Exception à afficher
```